### ISSUE TITLE
ENH: Allow Dockerfile input from stdio

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [projectatomic/buildah#886]
**Description**

This is a feature request/enhancement discussion.  The below is a IRC chat that I'd with @pixdrift the other night.  I wanted to capture the thoughts so we could discuss further here.

```
<pixdrift> not sure if it's been discussed, but has anyone raised buildah bud taking Dockerfile input from stdin?
<tsweeney> I don't recall a discussion on Dockerfile input from stdio, but I was buried in e-mails may have slipped my noggin.
<pixdrift> I just had a use case raised by a user, and thought it was worth mentioning
<tsweeney> It's not an issue/enhancement yet is it?
<pixdrift> they have a pipeline that constructs the Dockerfile from environmental information, and want to remove the temporary step of writing it to disk
<tsweeney> hmm, interesting.
<pixdrift> nope, was wondering if it had been discussed/dropped.. before I raise it
<pixdrift> so something like
<pixdrift> echo "<dockerfile contents>" | buildah bud --stdin
<pixdrift> the other alternative is to switch them to native buildah commands, but they want the option to output the file in some scenarios
```

@pixdrift feel free to add additional details or whack me if I mis-represented our discussion

**Comment by rhatdan** [comment]
@pixdrift @TomSweeneyRedHat Does

# buildah bud -f /dev/stdin ~
Work?

**Comment by TomSweeneyRedHat** [comment]
Yeah I tried a couple of things without luck.  After that thought I'd throw this up here for a discussion.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat The patch only checks size on regular files, since this is a pipe it will allow it to proceed in my patch.

**Comment by TomSweeneyRedHat** [comment]
Addressed by #888, closing
